packages/server/api/src/app/workers/engine-controller.ts

```diff
@@ -135,15 +136,14 @@ export const flowEngineWorker: FastifyPluginAsyncTypebox = async (app) => {
       tags: runDetails.tags ?? [],
     });
 
-    if (populatedRun.status === FlowRunStatus.RUNNING) {
-      return;
-    }
-
-    if (runDetails.status === FlowRunStatus.PAUSED) {
+    if (
+      runDetails.status === FlowRunStatus.PAUSED &&
+      !isNil(runDetails.pauseMetadata)
```

Comment:
how can it be null and paused at the same time? if it's paused, we have to have the pauseMetadata 🤔